integreat_cms/static/src/js/forms/autosave.ts

```diff
@@ -1,6 +1,19 @@
 import tinymce from "tinymce";
 import { getCsrfToken } from "../utils/csrf-token";
 
+const formatDate = (date: Date) => {
+    const options: Intl.DateTimeFormatOptions = {
+        day: "2-digit",
+        month: "long",
+        year: "numeric",
+        hour: "2-digit",
+        minute: "2-digit",
+    };
+
+    const formattedDate = new Intl.DateTimeFormat(document.documentElement.lang, options).format(date);
+    return formattedDate.replace(",", "");
```

Comment:
```suggestion
    return new Intl.DateTimeFormat(document.documentElement.lang, options).format(date);
```
Why were the commas removed? In the German date, there aren't any, and in the English date they're kind of important, right?

Reply:
Yes, I'm sorry. I removed it now